core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/ShaclSail.java

```diff
@@ -959,6 +959,17 @@ public IsolationLevel getDefaultIsolationLevel() {
 		return super.getDefaultIsolationLevel();
 	}
 
+	public boolean hasShapes() {
```

Comment:
Adding a public method makes an API change, which doesn't really fit with a patch release. But given that you're only using it from the ShaclSailConnection class you could just make it package-private I think.

Reply:
Thanks!